src/testers.jl

```diff
@@ -185,21 +185,13 @@ function frule_test(f, xẋs::Tuple{Any, Any}...; rtol=1e-9, atol=1e-9, fdm=_fdm
     xs, ẋs = first.(xẋs), last.(xẋs)
     Ω_ad, dΩ_ad = frule((NO_FIELDS, ẋs...), f, xs...; fkwargs...)
     Ω = f(xs...; fkwargs...)
-    # if equality check fails, check approximate equality
-    # use collect so can do vector equality
-    # TODO: add isapprox replacement that works for more types
     @test Ω_ad == Ω || isapprox(collect(Ω_ad), collect(Ω); rtol=rtol, atol=atol)
 
     ẋs_is_ignored = ẋs .== nothing
     # Correctness testing via finite differencing.
     dΩ_fd = _make_jvp_call(fdm, (xs...) -> f(xs...; fkwargs...), xs, ẋs, ẋs_is_ignored)
-    @test isapprox(
-        collect(extern.(dΩ_ad)),  # Use collect so can use vector equality
-        collect(dΩ_fd);
-        rtol=rtol,
-        atol=atol,
-        kwargs...
-    )
+    accumulant = rand_tangent(Ω)
```

Comment:
in `frule_test` we don't ask for a accumulant, so her we use the tangent generation stuff to do so.

In `rrule_test` we do ask for an accumulant.

Probably we should actually be asking for an accumulant for both of them
and have some `auto` flag that can be passed in to have one generated via `rand_tangent`.

Reply:
Can you add a comment explaining why it's called `accumulant`?

Reply:
remove